.readthedocs.yml

```diff
@@ -10,7 +10,7 @@ build:
       # and therefore tested in this no-CUDA environment
       - |
         pip install \
-          -C rapidsai.matrix-entry="cuda=12.2" \
+          -C rapidsai.matrix-entry="cuda=12.2;cuda_suffixed=true" \
```

Comment:
Was this missed in an upgrade to 12.5? Feel free to slip that in here or in a follow-up PR.

Reply:
Probably, yes. I'll put up a follow-up PR fixing this... against 24.10. There's not a functional difference for `ucx-py`, since this is just a way to matrix matrices with wildcard selectors like `"12.*"`.